### PR TITLE
[Vision Glass] Legal terms in phone UI

### PR DESCRIPTION
**Pull request by felipeerias**
Show the legal terms the first time that the user launches the Vision Glass app.

### CODE REVIEW

**Comment by felipeerias** [comment]
This PR depends on https://github.com/Igalia/wolvic/pull/1449 and https://github.com/Igalia/wolvic/pull/1460

File: app/src/visionglass/java/com/igalia/wolvic/FirstRunActivity.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    .setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE)
+                    .replace(R.id.fragment_placeholder, new PrivacyPolicyFragment())
+                    .commit();
+        } else {
+            // finish and go to the main activity
+            Intent intent = new Intent(this, VRBrowserActivity.class);
+            startActivity(intent);
+            finish();
+        }
+    }
```
Comment (svillar) [inline app/src/visionglass/java/com/igalia/wolvic/FirstRunActivity.java:56]: What is not clear to me is what happens if the user declines to accept either the TOS or the privacy policy. If I am not mistaken, in that case no shared preference will change and thus this FirstActivity won't ever finish.

File: app/src/visionglass/AndroidManifest.xml [hunk 2]
```diff
@@ -8,11 +8,17 @@
     <application>
         <activity
             android:name=".VRBrowserActivity"
-            android:screenOrientation="portrait">
+            android:screenOrientation="portrait"
+            android:exported="true">
```
Comment (svillar) [inline app/src/visionglass/AndroidManifest.xml:12]: Hmm, isn't this only for allowing external apps to call the activity? Shouldn't be needed for an activity from the same app id. Or maybe I am wrong

Comment (felipeerias) [reply app/src/visionglass/AndroidManifest.xml:12]: This fixes a warning, apparently the main activity should have `exported=true`.

File: app/src/visionglass/java/com/igalia/wolvic/FirstRunActivity.java (hunk 1 above)
Comment (felipeerias) [reply app/src/visionglass/java/com/igalia/wolvic/FirstRunActivity.java:56]: We are reusing the functionality that is already implemented in `FirstRunFragment.onDisagreeButtonClicked()`.

If you decline the terms we show a second dialog, and if you decline that one then we finish the application. This is the same behaviour that we have in the HVR flavour.

**Review by svillar — APPROVED** [review]

**Pull request merged**